ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml

```diff
@@ -21,8 +21,8 @@ tests:
     allow_skip_on_success: true
     cluster_profile: aws-perfscale
     env:
+      ADDITIONAL_WORKER_NODES: "117"
```

Comment:
I find this variable a little confusing as you might not know how many nodes are on the cluster currently. Can you have a variable instead that would take the number of total worker nodes you want, then in the command script it would find the number of worker nodes that are currently on the cluster and then pass the additional worker nodes to the run script?

Reply:
Interesting, but I do not see any added value on that. Every base cluster has 3 workers nodes by default. Also there isn't a single test in the CI where nodes get deployed without our knowledge. So I think it is better to keep the variable name as is, so that it makes it clear that there are additional nodes being added on the existing setup which is what justifies the purpose of this reusable step.